dev/dartdoc.dart

```diff
@@ -56,6 +56,22 @@ dependencies:
     args.add(name.substring(0, name.length - 5));
   }
 
+  [
```

Comment:
yup, other places in the script already depend on running from the root of the flutter repo